examples/algorithms/plot_girvan_newman.py

```diff
@@ -17,7 +17,8 @@
 
 # Load karate graph and find communities using Girvan-Newman
 G = nx.karate_club_graph()
-communities = list(nx.community.girvan_newman(G))
+communities_generator = nx.community.girvan_newman(G)
+communities = [tuple(sorted(c) for c in next(communities_generator)) for _ in range(4)]  # Getting the first 4 iterations
```

Comment:
I don't think these lines should change. Go ahead and compute the entire list.
Then in line 34 take out the `[0]` without adding the comment.

Thanks!